components/speedreader/resources/panel/components/lists/index.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react'
+
+import * as S from './style'
+import fontSerifSvg from '../../svg/fontSerif'
+import fontSansSvg from '../../svg/fontSans'
+import fontMonoSvg from '../../svg/fontMono'
+import fontDyslexicSvg from '../../svg/fontDyslexic'
+import contentTextOnlySvg from '../../svg/contentTextOnly'
+import contentTextWithImagesSvg from '../../svg/contentTextWithImages'
+
+const fontStyleOptions = [
+  {
+    title: 'Sans',
+    svgIcon: fontSansSvg
+  },
+  {
+    title: 'Serif',
+    svgIcon: fontSerifSvg
+  },
+  {
+    title: 'Mono',
+    svgIcon: fontMonoSvg
+  },
+  {
+    title: 'Dyslexic',
+    svgIcon: fontDyslexicSvg
+  }
+]
+
+const contentStyleOptions = [
+  {
+    title: 'Text with images',
+    svgIcon: contentTextWithImagesSvg
+  },
+  {
+    title: 'Text only',
+    svgIcon: contentTextOnlySvg
+  }
+]
+
+type OptionType = {
+  isSelected: boolean
+  children: JSX.Element
+  onClick?: Function
+  ariaLabel?: string
+}
+
+type ListBoxProps = {
+  children: JSX.Element | JSX.Element[]
+}
+
+function ListBox (props: ListBoxProps) {
+  return (
+    <S.Box role="listbox" aria-orientation="horizontal">
```

Comment:
Nice aria roles. Would be interested to see how easy it is to use vs radio / checkbox controls.